src/tsconfig.json

```diff
@@ -5,13 +5,18 @@
     "outDir": "../dist/",
     "inlineSources": true,
     "sourceMap": true,
-    "declaration": true,
+    "declaration": false,
```

Comment:
The lone casualty - I haven't been able to re-enable this setting yet for the hybrid state.